xsec/makeCrossSectionMCInputs.C

```diff
@@ -29,9 +29,9 @@ typedef Variable Var;
 typedef HadronVariable HVar;
```

Comment:
The ehad changes here aren't related to adler angle.

They could go directly into master next time.